paths in git and paths when publishing the code as a crate are slightly different.

diff --git a/tests/integration/snapshots/integration__events-fib-5.snap b/tests/integration/snapshots/integration__events-fib-5.snap
index 40555d5..458dd6b 100644
--- a/tests/integration/snapshots/integration__events-fib-5.snap
+++ b/tests/integration/snapshots/integration__events-fib-5.snap
@@ -10,7 +10,7 @@ expression: events
     target: "integration::fib"
     level: info
     module_path: "integration::fib"
-    file: tunnel/tests/integration/fib.rs
+    file: tests/integration/fib.rs
     line: 42
     fields:
       - approx
@@ -32,7 +32,7 @@ expression: events
     target: "integration::fib"
     level: warn
     module_path: "integration::fib"
-    file: tunnel/tests/integration/fib.rs
+    file: tests/integration/fib.rs
     line: 42
     fields:
       - message
@@ -51,7 +51,7 @@ expression: events
     target: fib
     level: info
     module_path: "integration::fib"
-    file: tunnel/tests/integration/fib.rs
+    file: tests/integration/fib.rs
     line: 42
     fields:
       - count
@@ -70,7 +70,7 @@ expression: events
     target: fib
     level: debug
     module_path: "integration::fib"
-    file: tunnel/tests/integration/fib.rs
+    file: tests/integration/fib.rs
     line: 42
     fields:
       - message
@@ -128,7 +128,7 @@ expression: events
     target: fib
     level: info
     module_path: "integration::fib"
-    file: tunnel/tests/integration/fib.rs
+    file: tests/integration/fib.rs
     line: 42
     fields:
       - return
@@ -148,7 +148,7 @@ expression: events
     target: "integration::fib"
     level: info
     module_path: "integration::fib"
-    file: tunnel/tests/integration/fib.rs
+    file: tests/integration/fib.rs
     line: 42
     fields:
       - message
